bin/wp-installer.js

```diff
@@ -0,0 +1,267 @@
+#!/usr/bin/env node
+
+const chalk = require('chalk');
+const output = require('./output');
+const emoji = require('node-emoji');
+const {exec} = require('promisify-child-process');
+const {execSync} = require('child_process');
+const files = require('./files');
+const ora = require('ora');
+
+const createDatabase = async(options) => {
+  const create = `mysql -u ${options.dbRootUser} --password=${options.dbRootPass} -e "CREATE DATABASE IF NOT EXISTS ${options.dbName}"`;
+  const grant = `mysql -u ${options.dbRootUser} --password=${options.dbRootPass} -e "GRANT ALL PRIVILEGES ON ${options.dbName}.* TO ${options.dbUser}@localhost IDENTIFIED BY 'wp';"`;
+
+  return new Promise((resolve, reject) => {
+    exec(create).then(() => {
+      exec(grant).then(() => {
+        resolve(true);
+      }).catch((errGrant) => {
+        reject(errGrant);
+      });
+    }).catch((errCreate) => {
+      reject(errCreate);
+    });
+  });
+};
+
+/**
+ * Outputs WP login info.
+ *
+ * @param string url   Website's local URL.
+ * @param string user  WP user.
+ * @param string pass  WP user's pass.
+ */
+const outputWPLoginInfo = (url, user, pass) => {
+  console.log('Your login info:');
+  console.log('-------------------');
+  console.log(`Login: ${chalk.green(url)}${chalk.green('/wp-admin')}`);
+  console.log(`User: ${chalk.green(user)}`);
+  console.log(`Pass: ${chalk.green(pass)}`);
+};
+
+/**
+ * Prompts the user for database name and returns it
+ *
+ * @return string
+ */
+const promptDatabase = () => output.prompt({
+  label: 'Please enter database name (make sure it exists and is empty):',
+  prompt: `${emoji.get('school_satchel')} Database name: `,
+  error: 'Database name cannot be empty',
+  required: true,
+}).trim();
+
+/**
+ * Prompts the user for database name and returns it
+ *
+ * @return string
+ */
+const promptVmDir = () => output.prompt({
+  label: '1. Please enter your project\'s \'vm_dir\' VVV path (by default it\'s /srv/www/PROJECTNAME/):',
```

Comment:
This line is misleading, remove the last `/` from the `PROJECTNAME` 😉 

I added the last line and I got error

```bash
Error: Command failed: vagrant ssh -- -t 'cd src/www/clients/test//public_html/;'
bash: line 0: cd: src/www/clients/test//public_html/: No such file or directory
```

Reply:
You're completely right, I just got so used to it I ignored the instructions. :) anyway, will change the instructions + add a trim function if the user adds the extra `/` anyway